Cargo.toml

```diff
@@ -18,7 +18,21 @@
 [package]
 name = "datasketches"
 version = "0.1.0"
+
 edition = "2024"
+rust-version = "1.85.0"
```

Comment:
edition 2024 requires at least MSRV=1.85.

I personally like moving MSRV eagerly because the old toolchain doesn't get maintained anyway. But other library authors would like to stay as low as possible to keep a wide user adoption range, which, in turn, blocks their dependencies from bumping MSRV.

This is another topic we may discuss later.